BrowserKit/Sources/Common/Theming/DarkTheme.swift

```diff
@@ -88,6 +102,7 @@ private struct DarkColourPalette: ThemeColourPalette {
     var borderAccentNonOpaque: UIColor = FXColors.Blue20.withAlphaComponent(0.2)
     var borderAccentPrivate: UIColor = FXColors.Purple60
     var borderInverted: UIColor = FXColors.DarkGrey90
+    var borderToolbarDivider: UIColor = FXColors.DarkGrey60
```

Comment:
That one confused me. It's placed under the `Layers` section in the [Figma file](https://www.figma.com/file/pEyGeE4KV5ytYHeXMfLcEr/Mobile-Styles?type=design&node-id=1594-15823&mode=design&t=Odj3QUw2Gn4L9Pnh-4), but then the name is `borderToolbarDivider`. Since it was starting with `border` I was searching it under the border section in Figma and not the layer one. Can we double-check if this is indeed a layer, or more a border?

Reply:
It should be under the Borders section, this was a mishap and crystal should be fixing this. There were some other discrepancies I found, which I documented the rest to be addressed in this ticket: https://mozilla-hub.atlassian.net/browse/FXIOS-7778

In regards to how our tokens are ordered, some members of the team chatted in coffee chat today about possibly having it sorted alphabetically. The design team is going to discuss this and I was going to follow up with the team on what we preferred after that conversation.

Reply:
Actually, the Figma file should be updated now! please verify again and let me know if any issues! @lmarceau

Reply:
Awesome thank you!